search/tests/test_indexing.py

```diff
@@ -36,3 +50,79 @@ def test_strip_xml(self):
         self.resource.save()
         self.assertTrue(self.count_results("you're it") == 1)
         self.assertTrue(self.count_results(xml) == 0)
+
+    def test_sorting(self):
+        """Test sorting for search"""
+        # remove the default resource to control the environment
+        self.resource.delete()
+        # create some resources
+        res1 = self.create_resource(**dict(
+            resource_type="example",
+            title="silly example 1",
+            content_xml="<blah>blah 1</blah>",
+            mpath="/blah1",
+            xa_nr_views=1001,
+            xa_nr_attempts=99,
+            xa_avg_grade=9.9
+        ))
+        res2 = self.create_resource(**dict(
```

Comment:
There are no tests where `res2` comes first. `res1` comes first once, and `res3` comes first twice.